Add job dependency: run > collate > postscript > sync. (#832)
payu-org/payu: docs/source/usage.rst

Postprocessing
==============

By default, payu submits all enabled postprocessing jobs automatically after the model run has completed.
Postprocessing includes 
- collate, 
- postscript, 
- sync. 
These jobs can also be run manually if required.
When payu submits postprocessing jobs, it adds job dependencies to ensure that they run in the correct order:: 

   collate -> postscript -> sync

If a postprocessing job failed, any subsequent job(s) will be terminated and removed from the queue.
You can use ``payu status`` to check the status of all postprocessing jobs.

> To clean up postprocessing jobs that have been killed, please run ``payu status --update`` once.

Reviewer: ```suggestion
Run ``payu status --update`` to ensure the status of killed jobs is correctly reported.
```

Reviewer: "Clean up" sounds a bit alarming. Am I right that it is just the `payu status` that would otherwise be affected? Or are there are actual files that need to be delete or moved?

Reviewer: Ok there is clean-up. I will remove this objection!

Author: The subsequent job files will still live in the file system, even though the server killed the job itself.
Running `payu status --update` will remove killed job files.
